operators/pkg/forge/containers.go

```diff
@@ -200,6 +207,19 @@ func MyDriveContainer(instance *clv1alpha2.Instance, opts *ContainerEnvOpts, mou
 	return mydriveContainer
 }
 
+// StandaloneContainer forges the Standalone application container of the environment.
+func StandaloneContainer(instance *clv1alpha2.Instance, environment *clv1alpha2.Environment, volumeMountPath string) corev1.Container {
+	standaloneContainer := AppContainer(instance, environment, volumeMountPath)
+	AddTCPPortToContainer(&standaloneContainer, GUIPortName, GUIPortNumber)
+
+	AddEnvVariableToContainer(&standaloneContainer, "CROWNLABS_BASE_URL", strings.ReplaceAll(instance.Status.URL, "https://", ""))
```

Comment:
Yes, it could be a good idea, especially because in some applications settings is necessary to have domain and path separated ( for example in jupyterlab ). Could be a good idea to specify 2 different environment variables (i.e CROWNLABS_DOMAIN and CROWNLABs_PATH)?

Reply:
Makes sense to me